indica/models/helike_spectrometer.py

```diff
@@ -383,6 +383,7 @@ def __call__(
         moment_analysis: bool = False,
         background: float = None,
         pixel_offset: int = None,
+        norm_y: xr.DataArray = None,
```

Comment:
why put this in the model? this seems something you'd like to have in a workflow, leaving the model as light as possible...

Reply:
I was toying around with not fitting the argon density since it tends to cause a lot of headache (I'm undecided if this is needed). I don't see how to feasibly do this outside of the model.

Reply:
I understand. 
But then wouldn't it be better to process the measured data e.g. subtract the background and normalize the w-line peak to 1?
Then you can impose background = 0 and the keyword norm_y could become simply normalize (or something similar) and simply measured_spectra/measured_spectra.max()?